charts/sysdig/README.md

```diff
@@ -125,7 +162,7 @@ $ helm install --namespace sysdig-agent sysdig-agent -f values.yaml sysdig/sysdi
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
 ## Node Image Analyzer
-
+ TODO Update
```

Comment:
We will still default to installing the NIA. Should we change the default to the NA? Should we remove the NIA entirely?

Reply:
I think we should promote the NA, perhaps we can add a major version bump to make this clear.

Reply:
Sounds good to me 👍